### ISSUE TITLE
nvim.interlace.rmd can't deal with Rprofile option browser as function

### ISSUE THREAD

**Issue by ltschmitt** [jalvesaq/Nvim-R#477]
Hi jalvesaq,
First of all, thanks for this wonderful plugin.

I recently changed my Rprofile from

`options(help_type = 'html', browser = 'chromium')`

to 

`options(help_type = 'html', browser = function(u) browseURL(url=u,browser=ifelse(grepl('127.0.0.1:.*/library/', u), 'st -e w3m', 'chromium')))`

this gives me the benifit of using w3m as my help browser, while still being able to view all my rendered Rmd files in the graphical browser.

The problem is that this causes an error in nvim.interlace.rmd():

`Error in paste0("ROpenDoc('", res, "', '", getOption("browser"), "')") :
  cannot coerce type 'closure' to vector of type 'character'
`

Basically its trying to paste the function, which is not possible. The problem occurs with \kr \kp \kh etc.
I looked at the nvim.interlace.rmd function, but I can't see how to adapt my Rprofile or the function to make this work.

**Comment by jalvesaq** [comment]
Please, update Nvim-R. It will now use the result of `getOption("browser")` only if it is a string. Otherwise, the html document will be opened by the default browser, called by `xdg-open`.

**Comment by ltschmitt** [comment]
Awesome, it works now, thanks!
